infra/base-images/base-builder/detect_repo.py

```diff
@@ -29,26 +29,37 @@
 
 
 def main():
-  """Function to get a git repos information based on its commit."""
+  """Function to get a git repos information based on its commit.
```

Comment:
`repo's`

Reply:
Also what information? Please elaborate.

Reply:
Done.

Reply:
I added this in the comments, but it gets the main repo's URL and name it is cloned to in the docker file.